File: core/src/main/java/com/graqr/threshr/model/redsky/product/VideoCaption.java
package com.graqr.threshr.model.redsky.product;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record VideoCaption(
        @JsonProperty("caption_url") String captionUrl,
        String language) {
}

File: src/main/kotlin/com/graqr/threshr/model/redsky/product/VideoCaption.kt
package com.graqr.threshr.model.redsky.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class VideoCaption(
    @param:JsonProperty("caption_url") val captionUrl: String, val language: String
) {}
